api/cloud_api/server.js

```diff
@@ -3,7 +3,8 @@ const { SceHttpServer } = require('../util/SceHttpServer');
 function main() {
   const API_ENDPOINTS = [
```

Comment:
you should be able to do 
```suggestion
  const API_ENDPOINTS =  __dirname + '/routes`;
```

and you wont need to keep adding to the array.